cpuinfo[core,tools]:arm64-osx -> 2022-07-19

Package: cpuinfo[core,tools]:arm64-osx -> 2022-07-19

**Host Environment**

- Host: arm64-osx
- Compiler: AppleClang 14.0.0.14000029
-    vcpkg-tool version: 2999-12-31-unknownhash
    vcpkg-scripts version: 7643561c2 2022-12-08 (24 hours ago)


**Failure logs**

```
-- Using cached pytorch-cpuinfo-5e63739504f0f8e18e941bd63b2d6d42536c7d90.tar.gz.
-- Extracting source /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/downloads/pytorch-cpuinfo-5e63739504f0f8e18e941bd63b2d6d42536c7d90.tar.gz
-- Using source at /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/cpuinfo/src/42536c7d90-2a15ca16d1.clean
-- Configuring arm64-osx
-- Building arm64-osx-dbg
-- Building arm64-osx-rel
-- Fixing pkgconfig file: /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/cpuinfo_arm64-osx/lib/pkgconfig/libcpuinfo.pc
-- Fixing pkgconfig file: /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/cpuinfo_arm64-osx/debug/lib/pkgconfig/libcpuinfo.pc
CMake Error at scripts/cmake/vcpkg_copy_tools.cmake:32 (message):
  Couldn't find tool "cpuid-dump":

      neither "/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/cpuinfo_arm64-osx/bin/cpuid-dump" nor "/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/packages/cpuinfo_arm64-osx/bin/cpuid-dump.app" exists
Call Stack (most recent call first):
  ports/cpuinfo/portfile.cmake:53 (vcpkg_copy_tools)
  scripts/ports.cmake:147 (include)



```

--- a/ports/cpuinfo/portfile.cmake
+++ b/ports/cpuinfo/portfile.cmake
@@ -50,8 +50,12 @@ vcpkg_fixup_pkgconfig() # pkg_check_modules(libcpuinfo)
 file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/debug/include")
 
 if("tools" IN_LIST FEATURES)
+    set(additional_tools "")
+    if(EXISTS "${CURRENT_PACKAGES_DIR}/bin/cpuid-dump")
+        list(APPEND additional_tools "cpuid-dump")
+    endif()
     vcpkg_copy_tools(
-        TOOL_NAMES cache-info cpuid-dump cpu-info isa-info
+        TOOL_NAMES cache-info cpu-info isa-info ${additional_tools}
         AUTO_CLEAN
     )
 endif()

--- a/ports/cpuinfo/vcpkg.json
+++ b/ports/cpuinfo/vcpkg.json
@@ -1,6 +1,7 @@
 {
   "name": "cpuinfo",
   "version-date": "2022-07-19",
+  "port-version": 1,
   "description": "CPU INFOrmation library (x86/x86-64/ARM/ARM64, Linux/Windows/Android/macOS/iOS)",
   "homepage": "https://github.com/pytorch/cpuinfo",
   "license": "BSD-2-Clause",

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -1726,7 +1726,7 @@
     },
     "cpuinfo": {
       "baseline": "2022-07-19",
-      "port-version": 0
+      "port-version": 1
     },
     "cr": {
       "baseline": "2020-04-26",

--- a/versions/c-/cpuinfo.json
+++ b/versions/c-/cpuinfo.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "fd82cda8111eabcbc6317c215c4ad58b9778e7e0",
+      "version-date": "2022-07-19",
+      "port-version": 1
+    },
+    {
       "git-tree": "5ee1eff9fe6cfa584b697e71ce7081a69ef2c51d",
       "version-date": "2022-07-19",
       "port-version": 0

